replica/table.cc

```diff
@@ -790,12 +790,11 @@ table::start() {
 
 future<>
 table::stop() {
-    if (_async_gate.is_closed()) {
+    if (_pending_generic_ops_phaser.is_closed()) {
```

Comment:
maybe it's a good opportunity to introduce a `table::is_stopped()` method that will hide the details from the callers.
right now the logic is too spread out and replicated.

Reply:
Good idea, will do.

Reply:
Added a `_stopped` flag instead.